virtue/virtue-base/kickoff_root.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# This script is ran by the root user; add any necessary
+# root commands that must be ran during startup such as
+# adding iptable rules.
+
+# Allow enough time for the network rules to be copied over
+sleep 10
+
+# Loop through the networkRules file and add them to the
+# iptable rule chains
+cat /etc/networkRules | while read line; do
```

Comment:
What happens if there's no `networkRules` file? If it crashes out of the script, the container won't start. You may want a check for the file around this with some more explicit error message if there is no file.

Reply:
If there is no networkRules file, cat will report an error like "cat: No such file networkRules" but the script won't crash, which I think is what we want. The script will continue and execute kickoff.sh.